compiler/src/iree/compiler/Codegen/Common/GPU/GPUReduceBankConflicts.cpp

```diff
@@ -74,8 +75,11 @@ namespace {
 /// be removed once the better solution is implemented.
 struct GPUReduceBankConflictsPass final
     : GPUReduceBankConflictsBase<GPUReduceBankConflictsPass> {
-  GPUReduceBankConflictsPass(int64_t paddingSizeBits)
-      : paddingSizeBits(paddingSizeBits) {}
+  GPUReduceBankConflictsPass(
+      int64_t paddingSizeBits,
+      std::function<LogicalResult(mlir::FunctionOpInterface)> filterFn)
```

Comment:
Would be good to avoid such functions. If we move to the new way of defining passes this wont be supportable.